README

```diff
@@ -14,8 +14,23 @@ things mentioned above.
 DEPENDENCIES:
   - GCC and binutils
   - GNU Bash
+  - GNU Awk
```

Comment:
Does it really need gawk, or just any posix awk? Does mawk work?

Reply:
I guess it would work with any awk. I've tried it with mawk and it works. Should I update it accordingly?

Reply:
yes please

Reply:
done!